### PR TITLE
Update Node.js readme to avoid confusion

### PR DESCRIPTION
**Pull request by fhinkel**
We want to make sure nobody confuses this repo with the main repo.

Ref: https://github.com/nodejs/node-v8/issues/6

Any feelings on whether it's OK to delete the original Readme? Should we keep it and just prefix it with the explanation?

### CODE REVIEW

**Comment by targos** [comment]
I think it's OK to delete the README but it has to be done in the update script, somehow. This branch is overwritten by it.

**Comment by targos** [comment]
What we can do is discuss here about the contents and I'll add a step to the script that replaces the readme with it.

**Comment by fhinkel** [comment]
Sounds good!

File: README.md [hunk 1]
```diff
@@ -1,518 +1,8 @@
-<p align="center">
-  <a href="https://nodejs.org/">
-    <img alt="Node.js" src="https://nodejs.org/static/images/logo-light.svg" width="400"/>
-  </a>
-</p>
-<p align="center">
-  <a title="CII Best Practices" href="https://bestpractices.coreinfrastructure.org/projects/29"><img src="https://bestpractices.coreinfrastructure.org/projects/29/badge"></a>
-</p>
+This is a mirror of [Node.js](https://github.com/nodejs/node) 
+with V8 Tip-of-Tree.
```
Comment (TimothyGu) [inline README.md]: I'd explicitly call this out as experimental:

> This is an automatically updated **experimental** version of Node.js with V8 Tip-of-Tree embedded.

LGTM either way though.

Comment (fhinkel) [inline README.md]: Thanks, that's a great suggestions.

**Comment by targos** [comment]
Thank you @fhinkel I landed this on a new branch called [`readme`](https://github.com/nodejs/node-v8/tree/readme) so we can use pull requests to update the text.
I'm changing the update script to get the contents from there right now.

**Pull request closed without merging**